NOTICES

```diff
@@ -1120,6 +1121,7 @@ stache 2.2.0 : BSD 3-clause "New" or "Revised" License
 static-extend 0.1.2 : MIT License
 statuses 1.5.0 : MIT License
 stm-chans 3.0.0.4 : BSD 3-clause "New" or "Revised" License
+stm-conduit 4.0.1 : BSD 3-clause "New" or "Revised" License
```

Comment:
this one is correct.

Reply:
looking at the latest bazel haskell scan
https://dev.azure.com/digitalasset/adadc18a-d7df-446a-aacb-86042c1619c6/_apis/build/builds/68179/logs/74

the conduit related stuff (including xml-conduit and tar-conduit and http-conduit and cereal-conduit) from the log it returns is for version 0.3.2, 1.3.4, 1.3.5 and 4.0.1.  are the other versions maybe transitives of xml-conduit and tar-conduit and conduit-extra and friends?

here are the final calculated dependencies
```
2021-01-29T06:07:16.2040169Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency retry/0.8.1.2
2021-01-29T06:07:16.2040838Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency stm-conduit/4.0.1
2021-01-29T06:07:16.2041458Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency megaparsec/9.0.1
2021-01-29T06:07:16.2042049Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-default/0.7.1.1
2021-01-29T06:07:16.2042660Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency base64-bytestring/1.1.0.0
2021-01-29T06:07:16.2043282Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency system-filepath/0.4.14
2021-01-29T06:07:16.2043863Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency syb/0.7.1
2021-01-29T06:07:16.2044471Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency exceptions/0.10.4
2021-01-29T06:07:16.2045050Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency yaml/0.11.5.0
2021-01-29T06:07:16.2045635Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency conduit/1.3.4
2021-01-29T06:07:16.2046223Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency monad-logger/0.3.36
2021-01-29T06:07:16.2046813Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency zip-archive/0.4.1
2021-01-29T06:07:16.2047416Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency lsp-test/0.11.0.7
2021-01-29T06:07:16.2048031Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency Decimal/0.5.1
2021-01-29T06:07:16.2048629Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency monad-control/1.0.2.3
2021-01-29T06:07:16.2049268Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency scientific/0.3.6.2
2021-01-29T06:07:16.2049894Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haskell-lsp-types/0.23.0.0
2021-01-29T06:07:16.2050526Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency aeson-extra/0.4.1.3
2021-01-29T06:07:16.2051130Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency aeson-pretty/0.8.8
2021-01-29T06:07:16.2051723Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency split/0.2.3.4
2021-01-29T06:07:16.2052309Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency fast-logger/3.0.2
2021-01-29T06:07:16.2052926Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tar-conduit/0.3.2
2021-01-29T06:07:16.2053541Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency extra/1.7.1
2021-01-29T06:07:16.2054128Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency clock/0.8
2021-01-29T06:07:16.2054728Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency recursion-schemes/5.2.1
2021-01-29T06:07:16.2055348Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty/1.2.3
2021-01-29T06:07:16.2055952Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency simple-smt/0.9.4
2021-01-29T06:07:16.2056568Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty-expected-failure/0.11.1.2
2021-01-29T06:07:16.2057215Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency regex-tdfa/1.3.1.0
2021-01-29T06:07:16.2057842Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hspec/2.7.5
2021-01-29T06:07:16.2058563Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty-quickcheck/0.10.1.2
2021-01-29T06:07:16.2059212Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency http-client-tls/0.3.5.3
2021-01-29T06:07:16.2059888Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency xml/1.3.14
2021-01-29T06:07:16.2060451Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency random/1.1
2021-01-29T06:07:16.2061020Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency either/5.0.1.1
2021-01-29T06:07:16.2061624Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unordered-containers/0.2.13.0
2021-01-29T06:07:16.2062245Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cryptohash/0.11.9
2021-01-29T06:07:16.2062831Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unix-compat/0.5.2
2021-01-29T06:07:16.2063421Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency conduit-extra/1.3.5
2021-01-29T06:07:16.2064016Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency dlist/0.8.0.8
2021-01-29T06:07:16.2064635Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unliftio/0.2.13.1
2021-01-29T06:07:16.2065211Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency path-io/1.6.2
2021-01-29T06:07:16.2065791Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency xml-conduit/1.9.0.0
2021-01-29T06:07:16.2066409Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency case-insensitive/1.2.1.0
2021-01-29T06:07:16.2067048Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haskell-lsp/0.23.0.0
2021-01-29T06:07:16.2067654Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ansi-wl-pprint/0.6.9
2021-01-29T06:07:16.2068263Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency monad-loops/0.4.3
2021-01-29T06:07:16.2068844Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cryptonite/0.27
2021-01-29T06:07:16.2069522Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency transformers-base/0.4.5.2
2021-01-29T06:07:16.2070136Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency vector/0.12.1.2
2021-01-29T06:07:16.2070717Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hlint/2.2.11
2021-01-29T06:07:16.2071286Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency async/2.2.2
2021-01-29T06:07:16.2071857Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency jwt/0.10.0
2021-01-29T06:07:16.2072442Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency aeson/1.5.5.1
2021-01-29T06:07:16.2073013Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency gitrev/1.3.1
2021-01-29T06:07:16.2073594Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency main-tester/0.2.0.1
2021-01-29T06:07:16.2074227Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty-ant-xml/1.1.7
2021-01-29T06:07:16.2074830Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unliftio-core/0.2.0.1
2021-01-29T06:07:16.2075427Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency file-embed/0.0.13.0
2021-01-29T06:07:16.2076027Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency blaze-html/0.9.1.2
2021-01-29T06:07:16.2076620Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hashable/1.3.0.0
2021-01-29T06:07:16.2077206Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency lifted-async/0.10.1.2
2021-01-29T06:07:16.2077792Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency memory/0.15.0
2021-01-29T06:07:16.2078425Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency network-uri/2.6.3.0
2021-01-29T06:07:16.2079016Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tls/1.5.4
2021-01-29T06:07:16.2079624Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency safe/0.3.19
2021-01-29T06:07:16.2080218Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency prettyprinter/1.7.0
2021-01-29T06:07:16.2081035Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency typed-process/0.2.6.0
2021-01-29T06:07:16.2081690Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency semver/0.4.0.1
2021-01-29T06:07:16.2082310Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency lifted-base/0.2.3.12
2021-01-29T06:07:16.2083072Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency terminal-progress-bar/0.4.1
2021-01-29T06:07:16.2083750Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency utf8-string/1.0.2
2021-01-29T06:07:16.2084436Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency resourcet/1.2.4.2
2021-01-29T06:07:16.2085027Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency nsis/0.3.3
2021-01-29T06:07:16.2085602Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cmark-gfm/0.2.2
2021-01-29T06:07:16.2086185Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency pretty-show/1.10
2021-01-29T06:07:16.2086781Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tar/0.5.1.1
2021-01-29T06:07:16.2087359Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency sorted-list/0.2.1.0
2021-01-29T06:07:16.2088101Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency network/3.1.1.1
2021-01-29T06:07:16.2088739Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency parser-combinators/1.2.1
2021-01-29T06:07:16.2089400Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency lens-aeson/1.1
2021-01-29T06:07:16.2089990Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency uri-encode/1.5.0.7
2021-01-29T06:07:16.2090583Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency connection/0.3.1
2021-01-29T06:07:16.2091178Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency temporary/1.3
2021-01-29T06:07:16.2091766Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty-golden/2.3.3.2
2021-01-29T06:07:16.2092349Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency path/0.7.0
2021-01-29T06:07:16.2092943Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tasty-hunit/0.10.0.3
2021-01-29T06:07:16.2093672Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency uniplate/1.6.13
2021-01-29T06:07:16.2094290Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency rope-utf16-splay/0.3.1.0
2021-01-29T06:07:16.2094900Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency http-conduit/2.3.7.4
2021-01-29T06:07:16.2095521Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haskell-src-exts/1.23.1
2021-01-29T06:07:16.2096126Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ansi-terminal/0.10.3
2021-01-29T06:07:16.2096712Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency filepattern/0.1.2
2021-01-29T06:07:16.2097306Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency silently/1.2.5.1
2021-01-29T06:07:16.2097944Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ghc-lib-parser-ex/8.8.5.8
2021-01-29T06:07:16.2098583Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optparse-applicative/0.15.1.0
2021-01-29T06:07:16.2099247Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency http-client/0.6.4.1
2021-01-29T06:07:16.2099841Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency proto3-wire/1.2.0
2021-01-29T06:07:16.2100470Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency safe-exceptions/0.1.7.1
2021-01-29T06:07:16.2101104Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ghc-lib-parser/8.8.1.20210101
2021-01-29T06:07:16.2101721Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency uuid/1.3.13
2021-01-29T06:07:16.2102297Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency repline/0.4.0.0
2021-01-29T06:07:16.2102907Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency lens/4.19.2
2021-01-29T06:07:16.2103513Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency http-types/0.12.3
2021-01-29T06:07:16.2104124Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency open-browser/0.2.1.0
2021-01-29T06:07:16.2104724Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency semigroupoids/5.3.5
2021-01-29T06:07:16.2105358Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tagged/0.8.6.1
2021-01-29T06:07:16.2105973Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency base16-bytestring/0.1.1.7
2021-01-29T06:07:16.2106564Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency stache/2.2.0
2021-01-29T06:07:16.2107144Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency stm-chans/3.0.0.4
2021-01-29T06:07:16.2107857Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency base-orphans/0.8.4
2021-01-29T06:07:16.2108442Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hscolour/1.24.4
2021-01-29T06:07:16.2109171Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency network-info/0.2.0.10
2021-01-29T06:07:16.2109784Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency libyaml/0.1.2
2021-01-29T06:07:16.2110364Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hslogger/1.3.1.0
2021-01-29T06:07:16.2110958Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cereal-conduit/0.8.0
2021-01-29T06:07:16.2111596Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency base-compat-batteries/0.11.2
2021-01-29T06:07:16.2112210Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency Diff/0.4.0
2021-01-29T06:07:16.2112783Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency turtle/1.5.20
2021-01-29T06:07:16.2113365Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cmdargs/0.10.20
2021-01-29T06:07:16.2113961Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency call-stack/0.2.0
2021-01-29T06:07:16.2114598Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency shake/0.18.5
2021-01-29T06:07:16.2115192Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency entropy/0.4.1.6
2021-01-29T06:07:16.2115781Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency wcwidth/0.0.2
2021-01-29T06:07:16.2116362Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency asn1-types/0.3.4
2021-01-29T06:07:16.2116955Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency attoparsec/0.13.2.4
2021-01-29T06:07:16.2117545Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency uuid-types/1.0.3
2021-01-29T06:07:16.2118195Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cryptohash-sha1/0.11.100.1
2021-01-29T06:07:16.2118848Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ghc-lib/8.8.1.20210101
2021-01-29T06:07:16.2119540Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency x509-store/1.6.7
2021-01-29T06:07:16.2120204Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency easy-file/0.2.2
2021-01-29T06:07:16.2120922Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency blaze-markup/0.8.2.7
2021-01-29T06:07:16.2121575Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-default-class/0.1.2.0
2021-01-29T06:07:16.2122217Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency auto-update/0.1.6
2021-01-29T06:07:16.2122822Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency integer-logarithms/1.0.3.1
2021-01-29T06:07:16.2123452Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hourglass/0.2.12
2021-01-29T06:07:16.2124076Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency kan-extensions/5.2.1
2021-01-29T06:07:16.2124694Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency generic-deriving/1.13.1
2021-01-29T06:07:16.2125297Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency blaze-builder/0.4.1.0
2021-01-29T06:07:16.2125913Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-fix/0.3.0
2021-01-29T06:07:16.2126536Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haskell-lexer/1.1
2021-01-29T06:07:16.2127130Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency parameterized/0.5.0.0
2021-01-29T06:07:16.2127711Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency socks/0.6.1
2021-01-29T06:07:16.2128339Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency proto3-suite/0.4.2.0
2021-01-29T06:07:16.2128938Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency xml-types/0.3.8
2021-01-29T06:07:16.2129559Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unix-time/0.4.7
2021-01-29T06:07:16.2130135Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency digest/0.0.1.2
2021-01-29T06:07:16.2130723Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cereal/0.5.8.1
2021-01-29T06:07:16.2131297Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency bifunctors/5.5.9
2021-01-29T06:07:16.2131874Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency strict/0.4.0.1
2021-01-29T06:07:16.2132625Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency contravariant/1.5.3
2021-01-29T06:07:16.2133224Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency free/5.1.5
2021-01-29T06:07:16.2133911Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-default-instances-dlist/0.0.1
2021-01-29T06:07:16.2134586Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cryptohash-md5/0.11.100.1
2021-01-29T06:07:16.2135195Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hspec-discover/2.7.5
2021-01-29T06:07:16.2135782Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency time-compat/1.9.5
2021-01-29T06:07:16.2136361Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency zlib/0.6.2.2
2021-01-29T06:07:16.2136964Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency conduit-parse/0.2.1.0
2021-01-29T06:07:16.2137568Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency th-abstraction/0.4.2.0
2021-01-29T06:07:16.2138176Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency transformers-compat/0.6.6
2021-01-29T06:07:16.2138830Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency conduit-combinators/1.3.0
2021-01-29T06:07:16.2139492Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency these/1.1.1.1
2021-01-29T06:07:16.2140094Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency primitive/0.7.1.0
2021-01-29T06:07:16.2140687Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency ghcide/0.1.0
2021-01-29T06:07:16.2141271Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency QuickCheck/2.14.2
2021-01-29T06:07:16.2141885Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency constraints/0.12
2021-01-29T06:07:16.2142476Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cpphs/1.20.9.1
2021-01-29T06:07:16.2143085Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hspec-expectations/0.8.2
2021-01-29T06:07:16.2143683Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency semialign/1.1.0.1
2021-01-29T06:07:16.2144313Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency profunctors/5.5.2
2021-01-29T06:07:16.2144940Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency mime-types/0.1.0.9
2021-01-29T06:07:16.2145534Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency terminal-size/0.3.2.1
2021-01-29T06:07:16.2146130Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency x509-system/1.6.6
2021-01-29T06:07:16.2146764Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency comonad/5.0.8
2021-01-29T06:07:16.2147379Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hspec-core/2.7.5
2021-01-29T06:07:16.2147971Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency regex-base/0.94.0.0
2021-01-29T06:07:16.2148546Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cookie/0.4.5
2021-01-29T06:07:16.2149212Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency aeson-compat/0.3.9
2021-01-29T06:07:16.2149809Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency Glob/0.10.1
2021-01-29T06:07:16.2150398Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency x509-validation/1.6.11
2021-01-29T06:07:16.2151007Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency parallel/3.2.2.0
2021-01-29T06:07:16.2151584Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency colour/2.3.5
2021-01-29T06:07:16.2152154Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency zip/1.5.0
2021-01-29T06:07:16.2152734Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency basement/0.0.11
2021-01-29T06:07:16.2153335Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency cabal-doctest/1.0.8
2021-01-29T06:07:16.2153944Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency unbounded-delays/0.1.1.1
2021-01-29T06:07:16.2154564Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency byteable/0.1.1
2021-01-29T06:07:16.2155155Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency x509/1.7.5
2021-01-29T06:07:16.2155739Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency asn1-encoding/0.9.6
2021-01-29T06:07:16.2156330Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency reflection/2.1.6
2021-01-29T06:07:16.2157032Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency refact/0.3.0.2
2021-01-29T06:07:16.2157638Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency attoparsec-iso8601/1.0.2.0
2021-01-29T06:07:16.2158351Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency mono-traversable/1.0.15.1
2021-01-29T06:07:16.2159002Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-default-instances-old-locale/0.0.1
2021-01-29T06:07:16.2159697Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency streaming-commons/0.2.2.1
2021-01-29T06:07:16.2160328Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency base-compat/0.11.2
2021-01-29T06:07:16.2161026Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency semigroups/0.19.1
2021-01-29T06:07:16.2161675Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-default-instances-containers/0.0.1
2021-01-29T06:07:16.2162299Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency distributive/0.6.2.1
2021-01-29T06:07:16.2162907Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency time-locale-compat/0.1.1.5
2021-01-29T06:07:16.2163516Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency invariant/0.5.4
2021-01-29T06:07:16.2164119Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haskell-src/1.0.3.1
2021-01-29T06:07:16.2164708Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency foldl/1.4.10
2021-01-29T06:07:16.2165308Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency quickcheck-instances/0.3.25.2
2021-01-29T06:07:16.2165955Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency insert-ordered-containers/0.2.3.1
2021-01-29T06:07:16.2166556Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency HUnit/1.6.1.0
2021-01-29T06:07:16.2167134Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency adjunctions/4.4
2021-01-29T06:07:16.2167753Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency neat-interpolation/0.5.1.2
2021-01-29T06:07:16.2168511Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency indexed-traversable/0.1.1
2021-01-29T06:07:16.2169264Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency tf-random/0.5
2021-01-29T06:07:16.2169909Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency swagger2/2.6
2021-01-29T06:07:16.2170513Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency vector-algorithms/0.8.0.4
2021-01-29T06:07:16.2171101Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hpp/0.6.3
2021-01-29T06:07:16.2171694Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optparse-generic/1.3.1
2021-01-29T06:07:16.2172309Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency asn1-parse/0.9.5
2021-01-29T06:07:16.2172881Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency assoc/1.0.2
2021-01-29T06:07:16.2173490Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency prettyprinter-ansi-terminal/1.1.2
2021-01-29T06:07:16.2174151Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency system-fileio/0.3.16.4
2021-01-29T06:07:16.2174749Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency heaps/0.3.6.1
2021-01-29T06:07:16.2175339Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency network-bsd/2.8.1.0
2021-01-29T06:07:16.2175937Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency pem/0.2.4
2021-01-29T06:07:16.2176520Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency old-locale/1.0.0.7
2021-01-29T06:07:16.2177116Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency quickcheck-io/0.2.0
2021-01-29T06:07:16.2177705Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency parsers/0.12.10
2021-01-29T06:07:16.2178316Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency managed/1.0.8
2021-01-29T06:07:16.2178936Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency language-c/0.8.3
2021-01-29T06:07:16.2179575Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency data-diverse/4.7.0.0
2021-01-29T06:07:16.2180179Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency splitmix/0.1.0.3
2021-01-29T06:07:16.2180759Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency setenv/0.1.1.3
2021-01-29T06:07:16.2181477Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency old-time/1.1.0.3
2021-01-29T06:07:16.2182069Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency range-set-list/0.1.3.1
2021-01-29T06:07:16.2182747Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency StateVar/1.2.1
2021-01-29T06:07:16.2183429Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optional-args/1.0.2
2021-01-29T06:07:16.2184042Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency fuzzy/0.1.0.0
2021-01-29T06:07:16.2184638Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency type-equality/1
2021-01-29T06:07:16.2185209Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency hostname/1.0
2021-01-29T06:07:16.2185786Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency polyparse/1.13
2021-01-29T06:07:16.2186384Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency haddock-library/1.9.0
2021-01-29T06:07:16.2186977Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optics-th/0.3.0.2
2021-01-29T06:07:16.2187564Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency http-media/0.8.0.0
2021-01-29T06:07:16.2188150Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency js-dgtable/0.5.2
2021-01-29T06:07:16.2188751Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optics-core/0.3.0.1
2021-01-29T06:07:16.2189382Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency generics-sop/0.5.1.0
2021-01-29T06:07:16.2190001Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency optics-extra/0.3
2021-01-29T06:07:16.2190606Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency void/0.7.3
2021-01-29T06:07:16.2191178Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency js-flot/0.8.3
2021-01-29T06:07:16.2191757Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency js-jquery/3.3.1
2021-01-29T06:07:16.2192386Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency mwc-random/0.14.0.0
2021-01-29T06:07:16.2192986Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency charset/0.3.7.1
2021-01-29T06:07:16.2193655Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency monoid-subclasses/1.0.1
2021-01-29T06:07:16.2194256Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency Only/0.1
2021-01-29T06:07:16.2194866Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency math-functions/0.3.4.1
2021-01-29T06:07:16.2195463Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency sop-core/0.5.0.1
2021-01-29T06:07:16.2196060Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency indexed-profunctors/0.1
2021-01-29T06:07:16.2196664Z 2021-01-29 06:07:11 DEBUG [main] --- Adding dependency primes/0.2.1.0
2021-01-29T06:07:16.2197338Z 2021-01-29 06:07:11 INFO  [main] --- Number of dependencies discovered for rule HASKELL_CABAL_LIBRARY: 255
2021-01-29T06:07:16.2276902Z 2021-01-29 06:07:11 DEBUG [main] --- Dependencies discovered for rule HASKELL_CABAL_LIBRARY: [{"externalId":{"forge":"hackage","pieces":[retry, 0.8.1.2],"prefix":null,"suffix":null},"name":"retry","version":"0.8.1.2"}, {"externalId":{"forge":"hackage","pieces":[stm-conduit, 4.0.1],"prefix":null,"suffix":null},"name":"stm-conduit","version":"4.0.1"}, {"externalId":{"forge":"hackage","pieces":[megaparsec, 9.0.1],"prefix":null,"suffix":null},"name":"megaparsec","version":"9.0.1"}, {"externalId":{"forge":"hackage","pieces":[data-default, 0.7.1.1],"prefix":null,"suffix":null},"name":"data-default","version":"0.7.1.1"}, {"externalId":{"forge":"hackage","pieces":[base64-bytestring, 1.1.0.0],"prefix":null,"suffix":null},"name":"base64-bytestring","version":"1.1.0.0"}, {"externalId":{"forge":"hackage","pieces":[system-filepath, 0.4.14],"prefix":null,"suffix":null},"name":"system-filepath","version":"0.4.14"}, {"externalId":{"forge":"hackage","pieces":[syb, 0.7.1],"prefix":null,"suffix":null},"name":"syb","version":"0.7.1"}, {"externalId":{"forge":"hackage","pieces":[exceptions, 0.10.4],"prefix":null,"suffix":null},"name":"exceptions","version":"0.10.4"}, {"externalId":{"forge":"hackage","pieces":[yaml, 0.11.5.0],"prefix":null,"suffix":null},"name":"yaml","version":"0.11.5.0"}, {"externalId":{"forge":"hackage","pieces":[conduit, 1.3.4],"prefix":null,"suffix":null},"name":"conduit","version":"1.3.4"}, {"externalId":{"forge":"hackage","pieces":[monad-logger, 0.3.36],"prefix":null,"suffix":null},"name":"monad-logger","version":"0.3.36"}, {"externalId":{"forge":"hackage","pieces":[zip-archive, 0.4.1],"prefix":null,"suffix":null},"name":"zip-archive","version":"0.4.1"}, {"externalId":{"forge":"hackage","pieces":[lsp-test, 0.11.0.7],"prefix":null,"suffix":null},"name":"lsp-test","version":"0.11.0.7"}, {"externalId":{"forge":"hackage","pieces":[Decimal, 0.5.1],"prefix":null,"suffix":null},"name":"Decimal","version":"0.5.1"}, {"externalId":{"forge":"hackage","pieces":[monad-control, 1.0.2.3],"prefix":null,"suffix":null},"name":"monad-control","version":"1.0.2.3"}, {"externalId":{"forge":"hackage","pieces":[scientific, 0.3.6.2],"prefix":null,"suffix":null},"name":"scientific","version":"0.3.6.2"}, {"externalId":{"forge":"hackage","pieces":[haskell-lsp-types, 0.23.0.0],"prefix":null,"suffix":null},"name":"haskell-lsp-types","version":"0.23.0.0"}, {"externalId":{"forge":"hackage","pieces":[aeson-extra, 0.4.1.3],"prefix":null,"suffix":null},"name":"aeson-extra","version":"0.4.1.3"}, {"externalId":{"forge":"hackage","pieces":[aeson-pretty, 0.8.8],"prefix":null,"suffix":null},"name":"aeson-pretty","version":"0.8.8"}, {"externalId":{"forge":"hackage","pieces":[split, 0.2.3.4],"prefix":null,"suffix":null},"name":"split","version":"0.2.3.4"}, {"externalId":{"forge":"hackage","pieces":[fast-logger, 3.0.2],"prefix":null,"suffix":null},"name":"fast-logger","version":"3.0.2"}, {"externalId":{"forge":"hackage","pieces":[tar-conduit, 0.3.2],"prefix":null,"suffix":null},"name":"tar-conduit","version":"0.3.2"}, {"externalId":{"forge":"hackage","pieces":[extra, 1.7.1],"prefix":null,"suffix":null},"name":"extra","version":"1.7.1"}, {"externalId":{"forge":"hackage","pieces":[clock, 0.8],"prefix":null,"suffix":null},"name":"clock","version":"0.8"}, {"externalId":{"forge":"hackage","pieces":[recursion-schemes, 5.2.1],"prefix":null,"suffix":null},"name":"recursion-schemes","version":"5.2.1"}, {"externalId":{"forge":"hackage","pieces":[tasty, 1.2.3],"prefix":null,"suffix":null},"name":"tasty","version":"1.2.3"}, {"externalId":{"forge":"hackage","pieces":[simple-smt, 0.9.4],"prefix":null,"suffix":null},"name":"simple-smt","version":"0.9.4"}, {"externalId":{"forge":"hackage","pieces":[tasty-expected-failure, 0.11.1.2],"prefix":null,"suffix":null},"name":"tasty-expected-failure","version":"0.11.1.2"}, {"externalId":{"forge":"hackage","pieces":[regex-tdfa, 1.3.1.0],"prefix":null,"suffix":null},"name":"regex-tdfa","version":"1.3.1.0"}, {"externalId":{"forge":"hackage","pieces":[hspec, 2.7.5],"prefix":null,"suffix":null},"name":"hspec","version":"2.7.5"}, {"externalId":{"forge":"hackage","pieces":[tasty-quickcheck, 0.10.1.2],"prefix":null,"suffix":null},"name":"tasty-quickcheck","version":"0.10.1.2"}, {"externalId":{"forge":"hackage","pieces":[http-client-tls, 0.3.5.3],"prefix":null,"suffix":null},"name":"http-client-tls","version":"0.3.5.3"}, {"externalId":{"forge":"hackage","pieces":[xml, 1.3.14],"prefix":null,"suffix":null},"name":"xml","version":"1.3.14"}, {"externalId":{"forge":"hackage","pieces":[random, 1.1],"prefix":null,"suffix":null},"name":"random","version":"1.1"}, {"externalId":{"forge":"hackage","pieces":[either, 5.0.1.1],"prefix":null,"suffix":null},"name":"either","version":"5.0.1.1"}, {"externalId":{"forge":"hackage","pieces":[unordered-containers, 0.2.13.0],"prefix":null,"suffix":null},"name":"unordered-containers","version":"0.2.13.0"}, {"externalId":{"forge":"hackage","pieces":[cryptohash, 0.11.9],"prefix":null,"suffix":null},"name":"cryptohash","version":"0.11.9"}, {"externalId":{"forge":"hackage","pieces":[unix-compat, 0.5.2],"prefix":null,"suffix":null},"name":"unix-compat","version":"0.5.2"}, {"externalId":{"forge":"hackage","pieces":[conduit-extra, 1.3.5],"prefix":null,"suffix":null},"name":"conduit-extra","version":"1.3.5"}, {"externalId":{"forge":"hackage","pieces":[dlist, 0.8.0.8],"prefix":null,"suffix":null},"name":"dlist","version":"0.8.0.8"}, {"externalId":{"forge":"hackage","pieces":[unliftio, 0.2.13.1],"prefix":null,"suffix":null},"name":"unliftio","version":"0.2.13.1"}, {"externalId":{"forge":"hackage","pieces":[path-io, 1.6.2],"prefix":null,"suffix":null},"name":"path-io","version":"1.6.2"}, {"externalId":{"forge":"hackage","pieces":[xml-conduit, 1.9.0.0],"prefix":null,"suffix":null},"name":"xml-conduit","version":"1.9.0.0"}, {"externalId":{"forge":"hackage","pieces":[case-insensitive, 1.2.1.0],"prefix":null,"suffix":null},"name":"case-insensitive","version":"1.2.1.0"}, {"externalId":{"forge":"hackage","pieces":[haskell-lsp, 0.23.0.0],"prefix":null,"suffix":null},"name":"haskell-lsp","version":"0.23.0.0"}, {"externalId":{"forge":"hackage","pieces":[ansi-wl-pprint, 0.6.9],"prefix":null,"suffix":null},"name":"ansi-wl-pprint","version":"0.6.9"}, {"externalId":{"forge":"hackage","pieces":[monad-loops, 0.4.3],"prefix":null,"suffix":null},"name":"monad-loops","version":"0.4.3"}, {"externalId":{"forge":"hackage","pieces":[cryptonite, 0.27],"prefix":null,"suffix":null},"name":"cryptonite","version":"0.27"}, {"externalId":{"forge":"hackage","pieces":[transformers-base, 0.4.5.2],"prefix":null,"suffix":null},"name":"transformers-base","version":"0.4.5.2"}, {"externalId":{"forge":"hackage","pieces":[vector, 0.12.1.2],"prefix":null,"suffix":null},"name":"vector","version":"0.12.1.2"}, {"externalId":{"forge":"hackage","pieces":[hlint, 2.2.11],"prefix":null,"suffix":null},"name":"hlint","version":"2.2.11"}, {"externalId":{"forge":"hackage","pieces":[async, 2.2.2],"prefix":null,"suffix":null},"name":"async","version":"2.2.2"}, {"externalId":{"forge":"hackage","pieces":[jwt, 0.10.0],"prefix":null,"suffix":null},"name":"jwt","version":"0.10.0"}, {"externalId":{"forge":"hackage","pieces":[aeson, 1.5.5.1],"prefix":null,"suffix":null},"name":"aeson","version":"1.5.5.1"}, {"externalId":{"forge":"hackage","pieces":[gitrev, 1.3.1],"prefix":null,"suffix":null},"name":"gitrev","version":"1.3.1"}, {"externalId":{"forge":"hackage","pieces":[main-tester, 0.2.0.1],"prefix":null,"suffix":null},"name":"main-tester","version":"0.2.0.1"}, {"externalId":{"forge":"hackage","pieces":[tasty-ant-xml, 1.1.7],"prefix":null,"suffix":null},"name":"tasty-ant-xml","version":"1.1.7"}, {"externalId":{"forge":"hackage","pieces":[unliftio-core, 0.2.0.1],"prefix":null,"suffix":null},"name":"unliftio-core","version":"0.2.0.1"}, {"externalId":{"forge":"hackage","pieces":[file-embed, 0.0.13.0],"prefix":null,"suffix":null},"name":"file-embed","version":"0.0.13.0"}, {"externalId":{"forge":"hackage","pieces":[blaze-html, 0.9.1.2],"prefix":null,"suffix":null},"name":"blaze-html","version":"0.9.1.2"}, {"externalId":{"forge":"hackage","pieces":[hashable, 1.3.0.0],"prefix":null,"suffix":null},"name":"hashable","version":"1.3.0.0"}, {"externalId":{"forge":"hackage","pieces":[lifted-async, 0.10.1.2],"prefix":null,"suffix":null},"name":"lifted-async","version":"0.10.1.2"}, {"externalId":{"forge":"hackage","pieces":[memory, 0.15.0],"prefix":null,"suffix":null},"name":"memory","version":"0.15.0"}, {"externalId":{"forge":"hackage","pieces":[network-uri, 2.6.3.0],"prefix":null,"suffix":null},"name":"network-uri","version":"2.6.3.0"}, {"externalId":{"forge":"hackage","pieces":[tls, 1.5.4],"prefix":null,"suffix":null},"name":"tls","version":"1.5.4"}, {"externalId":{"forge":"hackage","pieces":[safe, 0.3.19],"prefix":null,"suffix":null},"name":"safe","version":"0.3.19"}, {"externalId":{"forge":"hackage","pieces":[prettyprinter, 1.7.0],"prefix":null,"suffix":null},"name":"prettyprinter","version":"1.7.0"}, {"externalId":{"forge":"hackage","pieces":[typed-process, 0.2.6.0],"prefix":null,"suffix":null},"name":"typed-process","version":"0.2.6.0"}, {"externalId":{"forge":"hackage","pieces":[semver, 0.4.0.1],"prefix":null,"suffix":null},"name":"semver","version":"0.4.0.1"}, {"externalId":{"forge":"hackage","pieces":[lifted-base, 0.2.3.12],"prefix":null,"suffix":null},"name":"lifted-base","version":"0.2.3.12"}, {"externalId":{"forge":"hackage","pieces":[terminal-progress-bar, 0.4.1],"prefix":null,"suffix":null},"name":"terminal-progress-bar","version":"0.4.1"}, {"externalId":{"forge":"hackage","pieces":[utf8-string, 1.0.2],"prefix":null,"suffix":null},"name":"utf8-string","version":"1.0.2"}, {"externalId":{"forge":"hackage","pieces":[resourcet, 1.2.4.2],"prefix":null,"suffix":null},"name":"resourcet","version":"1.2.4.2"}, {"externalId":{"forge":"hackage","pieces":[nsis, 0.3.3],"prefix":null,"suffix":null},"name":"nsis","version":"0.3.3"}, {"externalId":{"forge":"hackage","pieces":[cmark-gfm, 0.2.2],"prefix":null,"suffix":null},"name":"cmark-gfm","version":"0.2.2"}, {"externalId":{"forge":"hackage","pieces":[pretty-show, 1.10],"prefix":null,"suffix":null},"name":"pretty-show","version":"1.10"}, {"externalId":{"forge":"hackage","pieces":[tar, 0.5.1.1],"prefix":null,"suffix":null},"name":"tar","version":"0.5.1.1"}, {"externalId":{"forge":"hackage","pieces":[sorted-list, 0.2.1.0],"prefix":null,"suffix":null},"name":"sorted-list","version":"0.2.1.0"}, {"externalId":{"forge":"hackage","pieces":[network, 3.1.1.1],"prefix":null,"suffix":null},"name":"network","version":"3.1.1.1"}, {"externalId":{"forge":"hackage","pieces":[parser-combinators, 1.2.1],"prefix":null,"suffix":null},"name":"parser-combinators","version":"1.2.1"}, {"externalId":{"forge":"hackage","pieces":[lens-aeson, 1.1],"prefix":null,"suffix":null},"name":"lens-aeson","version":"1.1"}, {"externalId":{"forge":"hackage","pieces":[uri-encode, 1.5.0.7],"prefix":null,"suffix":null},"name":"uri-encode","version":"1.5.0.7"}, {"externalId":{"forge":"hackage","pieces":[connection, 0.3.1],"prefix":null,"suffix":null},"name":"connection","version":"0.3.1"}, {"externalId":{"forge":"hackage","pieces":[temporary, 1.3],"prefix":null,"suffix":null},"name":"temporary","version":"1.3"}, {"externalId":{"forge":"hackage","pieces":[tasty-golden, 2.3.3.2],"prefix":null,"suffix":null},"name":"tasty-golden","version":"2.3.3.2"}, {"externalId":{"forge":"hackage","pieces":[path, 0.7.0],"prefix":null,"suffix":null},"name":"path","version":"0.7.0"}, {"externalId":{"forge":"hackage","pieces":[tasty-hunit, 0.10.0.3],"prefix":null,"suffix":null},"name":"tasty-hunit","version":"0.10.0.3"}, {"externalId":{"forge":"hackage","pieces":[uniplate, 1.6.13],"prefix":null,"suffix":null},"name":"uniplate","version":"1.6.13"}, {"externalId":{"forge":"hackage","pieces":[rope-utf16-splay, 0.3.1.0],"prefix":null,"suffix":null},"name":"rope-utf16-splay","version":"0.3.1.0"}, {"externalId":{"forge":"hackage","pieces":[http-conduit, 2.3.7.4],"prefix":null,"suffix":null},"name":"http-conduit","version":"2.3.7.4"}, {"externalId":{"forge":"hackage","pieces":[haskell-src-exts, 1.23.1],"prefix":null,"suffix":null},"name":"haskell-src-exts","version":"1.23.1"}, {"externalId":{"forge":"hackage","pieces":[ansi-terminal, 0.10.3],"prefix":null,"suffix":null},"name":"ansi-terminal","version":"0.10.3"}, {"externalId":{"forge":"hackage","pieces":[filepattern, 0.1.2],"prefix":null,"suffix":null},"name":"filepattern","version":"0.1.2"}, {"externalId":{"forge":"hackage","pieces":[silently, 1.2.5.1],"prefix":null,"suffix":null},"name":"silently","version":"1.2.5.1"}, {"externalId":{"forge":"hackage","pieces":[ghc-lib-parser-ex, 8.8.5.8],"prefix":null,"suffix":null},"name":"ghc-lib-parser-ex","version":"8.8.5.8"}, {"externalId":{"forge":"hackage","pieces":[optparse-applicative, 0.15.1.0],"prefix":null,"suffix":null},"name":"optparse-applicative","version":"0.15.1.0"}, {"externalId":{"forge":"hackage","pieces":[http-client, 0.6.4.1],"prefix":null,"suffix":null},"name":"http-client","version":"0.6.4.1"}, {"externalId":{"forge":"hackage","pieces":[proto3-wire, 1.2.0],"prefix":null,"suffix":null},"name":"proto3-wire","version":"1.2.0"}, {"externalId":{"forge":"hackage","pieces":[safe-exceptions, 0.1.7.1],"prefix":null,"suffix":null},"name":"safe-exceptions","version":"0.1.7.1"}, {"externalId":{"forge":"hackage","pieces":[ghc-lib-parser, 8.8.1.20210101],"prefix":null,"suffix":null},"name":"ghc-lib-parser","version":"8.8.1.20210101"}, {"externalId":{"forge":"hackage","pieces":[uuid, 1.3.13],"prefix":null,"suffix":null},"name":"uuid","version":"1.3.13"}, {"externalId":{"forge":"hackage","pieces":[repline, 0.4.0.0],"prefix":null,"suffix":null},"name":"repline","version":"0.4.0.0"}, {"externalId":{"forge":"hackage","pieces":[lens, 4.19.2],"prefix":null,"suffix":null},"name":"lens","version":"4.19.2"}, {"externalId":{"forge":"hackage","pieces":[http-types, 0.12.3],"prefix":null,"suffix":null},"name":"http-types","version":"0.12.3"}, {"externalId":{"forge":"hackage","pieces":[open-browser, 0.2.1.0],"prefix":null,"suffix":null},"name":"open-browser","version":"0.2.1.0"}, {"externalId":{"forge":"hackage","pieces":[semigroupoids, 5.3.5],"prefix":null,"suffix":null},"name":"semigroupoids","version":"5.3.5"}, {"externalId":{"forge":"hackage","pieces":[tagged, 0.8.6.1],"prefix":null,"suffix":null},"name":"tagged","version":"0.8.6.1"}, {"externalId":{"forge":"hackage","pieces":[base16-bytestring, 0.1.1.7],"prefix":null,"suffix":null},"name":"base16-bytestring","version":"0.1.1.7"}, {"externalId":{"forge":"hackage","pieces":[stache, 2.2.0],"prefix":null,"suffix":null},"name":"stache","version":"2.2.0"}, {"externalId":{"forge":"hackage","pieces":[stm-chans, 3.0.0.4],"prefix":null,"suffix":null},"name":"stm-chans","version":"3.0.0.4"}, {"externalId":{"forge":"hackage","pieces":[base-orphans, 0.8.4],"prefix":null,"suffix":null},"name":"base-orphans","version":"0.8.4"}, {"externalId":{"forge":"hackage","pieces":[hscolour, 1.24.4],"prefix":null,"suffix":null},"name":"hscolour","version":"1.24.4"}, {"externalId":{"forge":"hackage","pieces":[network-info, 0.2.0.10],"prefix":null,"suffix":null},"name":"network-info","version":"0.2.0.10"}, {"externalId":{"forge":"hackage","pieces":[libyaml, 0.1.2],"prefix":null,"suffix":null},"name":"libyaml","version":"0.1.2"}, {"externalId":{"forge":"hackage","pieces":[hslogger, 1.3.1.0],"prefix":null,"suffix":null},"name":"hslogger","version":"1.3.1.0"}, {"externalId":{"forge":"hackage","pieces":[cereal-conduit, 0.8.0],"prefix":null,"suffix":null},"name":"cereal-conduit","version":"0.8.0"}, {"externalId":{"forge":"hackage","pieces":[base-compat-batteries, 0.11.2],"prefix":null,"suffix":null},"name":"base-compat-batteries","version":"0.11.2"}, {"externalId":{"forge":"hackage","pieces":[Diff, 0.4.0],"prefix":null,"suffix":null},"name":"Diff","version":"0.4.0"}, {"externalId":{"forge":"hackage","pieces":[turtle, 1.5.20],"prefix":null,"suffix":null},"name":"turtle","version":"1.5.20"}, {"externalId":{"forge":"hackage","pieces":[cmdargs, 0.10.20],"prefix":null,"suffix":null},"name":"cmdargs","version":"0.10.20"}, {"externalId":{"forge":"hackage","pieces":[call-stack, 0.2.0],"prefix":null,"suffix":null},"name":"call-stack","version":"0.2.0"}, {"externalId":{"forge":"hackage","pieces":[shake, 0.18.5],"prefix":null,"suffix":null},"name":"shake","version":"0.18.5"}, {"externalId":{"forge":"hackage","pieces":[entropy, 0.4.1.6],"prefix":null,"suffix":null},"name":"entropy","version":"0.4.1.6"}, {"externalId":{"forge":"hackage","pieces":[wcwidth, 0.0.2],"prefix":null,"suffix":null},"name":"wcwidth","version":"0.0.2"}, {"externalId":{"forge":"hackage","pieces":[asn1-types, 0.3.4],"prefix":null,"suffix":null},"name":"asn1-types","version":"0.3.4"}, {"externalId":{"forge":"hackage","pieces":[attoparsec, 0.13.2.4],"prefix":null,"suffix":null},"name":"attoparsec","version":"0.13.2.4"}, {"externalId":{"forge":"hackage","pieces":[uuid-types, 1.0.3],"prefix":null,"suffix":null},"name":"uuid-types","version":"1.0.3"}, {"externalId":{"forge":"hackage","pieces":[cryptohash-sha1, 0.11.100.1],"prefix":null,"suffix":null},"name":"cryptohash-sha1","version":"0.11.100.1"}, {"externalId":{"forge":"hackage","pieces":[ghc-lib, 8.8.1.20210101],"prefix":null,"suffix":null},"name":"ghc-lib","version":"8.8.1.20210101"}, {"externalId":{"forge":"hackage","pieces":[x509-store, 1.6.7],"prefix":null,"suffix":null},"name":"x509-store","version":"1.6.7"}, {"externalId":{"forge":"hackage","pieces":[easy-file, 0.2.2],"prefix":null,"suffix":null},"name":"easy-file","version":"0.2.2"}, {"externalId":{"forge":"hackage","pieces":[blaze-markup, 0.8.2.7],"prefix":null,"suffix":null},"name":"blaze-markup","version":"0.8.2.7"}, {"externalId":{"forge":"hackage","pieces":[data-default-class, 0.1.2.0],"prefix":null,"suffix":null},"name":"data-default-class","version":"0.1.2.0"}, {"externalId":{"forge":"hackage","pieces":[auto-update, 0.1.6],"prefix":null,"suffix":null},"name":"auto-update","version":"0.1.6"}, {"externalId":{"forge":"hackage","pieces":[integer-logarithms, 1.0.3.1],"prefix":null,"suffix":null},"name":"integer-logarithms","version":"1.0.3.1"}, {"externalId":{"forge":"hackage","pieces":[hourglass, 0.2.12],"prefix":null,"suffix":null},"name":"hourglass","version":"0.2.12"}, {"externalId":{"forge":"hackage","pieces":[kan-extensions, 5.2.1],"prefix":null,"suffix":null},"name":"kan-extensions","version":"5.2.1"}, {"externalId":{"forge":"hackage","pieces":[generic-deriving, 1.13.1],"prefix":null,"suffix":null},"name":"generic-deriving","version":"1.13.1"}, {"externalId":{"forge":"hackage","pieces":[blaze-builder, 0.4.1.0],"prefix":null,"suffix":null},"name":"blaze-builder","version":"0.4.1.0"}, {"externalId":{"forge":"hackage","pieces":[data-fix, 0.3.0],"prefix":null,"suffix":null},"name":"data-fix","version":"0.3.0"}, {"externalId":{"forge":"hackage","pieces":[haskell-lexer, 1.1],"prefix":null,"suffix":null},"name":"haskell-lexer","version":"1.1"}, {"externalId":{"forge":"hackage","pieces":[parameterized, 0.5.0.0],"prefix":null,"suffix":null},"name":"parameterized","version":"0.5.0.0"}, {"externalId":{"forge":"hackage","pieces":[socks, 0.6.1],"prefix":null,"suffix":null},"name":"socks","version":"0.6.1"}, {"externalId":{"forge":"hackage","pieces":[proto3-suite, 0.4.2.0],"prefix":null,"suffix":null},"name":"proto3-suite","version":"0.4.2.0"}, {"externalId":{"forge":"hackage","pieces":[xml-types, 0.3.8],"prefix":null,"suffix":null},"name":"xml-types","version":"0.3.8"}, {"externalId":{"forge":"hackage","pieces":[unix-time, 0.4.7],"prefix":null,"suffix":null},"name":"unix-time","version":"0.4.7"}, {"externalId":{"forge":"hackage","pieces":[digest, 0.0.1.2],"prefix":null,"suffix":null},"name":"digest","version":"0.0.1.2"}, {"externalId":{"forge":"hackage","pieces":[cereal, 0.5.8.1],"prefix":null,"suffix":null},"name":"cereal","version":"0.5.8.1"}, {"externalId":{"forge":"hackage","pieces":[bifunctors, 5.5.9],"prefix":null,"suffix":null},"name":"bifunctors","version":"5.5.9"}, {"externalId":{"forge":"hackage","pieces":[strict, 0.4.0.1],"prefix":null,"suffix":null},"name":"strict","version":"0.4.0.1"}, {"externalId":{"forge":"hackage","pieces":[contravariant, 1.5.3],"prefix":null,"suffix":null},"name":"contravariant","version":"1.5.3"}, {"externalId":{"forge":"hackage","pieces":[free, 5.1.5],"prefix":null,"suffix":null},"name":"free","version":"5.1.5"}, {"externalId":{"forge":"hackage","pieces":[data-default-instances-dlist, 0.0.1],"prefix":null,"suffix":null},"name":"data-default-instances-dlist","version":"0.0.1"}, {"externalId":{"forge":"hackage","pieces":[cryptohash-md5, 0.11.100.1],"prefix":null,"suffix":null},"name":"cryptohash-md5","version":"0.11.100.1"}, {"externalId":{"forge":"hackage","pieces":[hspec-discover, 2.7.5],"prefix":null,"suffix":null},"name":"hspec-discover","version":"2.7.5"}, {"externalId":{"forge":"hackage","pieces":[time-compat, 1.9.5],"prefix":null,"suffix":null},"name":"time-compat","version":"1.9.5"}, {"externalId":{"forge":"hackage","pieces":[zlib, 0.6.2.2],"prefix":null,"suffix":null},"name":"zlib","version":"0.6.2.2"}, {"externalId":{"forge":"hackage","pieces":[conduit-parse, 0.2.1.0],"prefix":null,"suffix":null},"name":"conduit-parse","version":"0.2.1.0"}, {"externalId":{"forge":"hackage","pieces":[th-abstraction, 0.4.2.0],"prefix":null,"suffix":null},"name":"th-abstraction","version":"0.4.2.0"}, {"externalId":{"forge":"hackage","pieces":[transformers-compat, 0.6.6],"prefix":null,"suffix":null},"name":"transformers-compat","version":"0.6.6"}, {"externalId":{"forge":"hackage","pieces":[conduit-combinators, 1.3.0],"prefix":null,"suffix":null},"name":"conduit-combinators","version":"1.3.0"}, {"externalId":{"forge":"hackage","pieces":[these, 1.1.1.1],"prefix":null,"suffix":null},"name":"these","version":"1.1.1.1"}, {"externalId":{"forge":"hackage","pieces":[primitive, 0.7.1.0],"prefix":null,"suffix":null},"name":"primitive","version":"0.7.1.0"}, {"externalId":{"forge":"hackage","pieces":[ghcide, 0.1.0],"prefix":null,"suffix":null},"name":"ghcide","version":"0.1.0"}, {"externalId":{"forge":"hackage","pieces":[QuickCheck, 2.14.2],"prefix":null,"suffix":null},"name":"QuickCheck","version":"2.14.2"}, {"externalId":{"forge":"hackage","pieces":[constraints, 0.12],"prefix":null,"suffix":null},"name":"constraints","version":"0.12"}, {"externalId":{"forge":"hackage","pieces":[cpphs, 1.20.9.1],"prefix":null,"suffix":null},"name":"cpphs","version":"1.20.9.1"}, {"externalId":{"forge":"hackage","pieces":[hspec-expectations, 0.8.2],"prefix":null,"suffix":null},"name":"hspec-expectations","version":"0.8.2"}, {"externalId":{"forge":"hackage","pieces":[semialign, 1.1.0.1],"prefix":null,"suffix":null},"name":"semialign","version":"1.1.0.1"}, {"externalId":{"forge":"hackage","pieces":[profunctors, 5.5.2],"prefix":null,"suffix":null},"name":"profunctors","version":"5.5.2"}, {"externalId":{"forge":"hackage","pieces":[mime-types, 0.1.0.9],"prefix":null,"suffix":null},"name":"mime-types","version":"0.1.0.9"}, {"externalId":{"forge":"hackage","pieces":[terminal-size, 0.3.2.1],"prefix":null,"suffix":null},"name":"terminal-size","version":"0.3.2.1"}, {"externalId":{"forge":"hackage","pieces":[x509-system, 1.6.6],"prefix":null,"suffix":null},"name":"x509-system","version":"1.6.6"}, {"externalId":{"forge":"hackage","pieces":[comonad, 5.0.8],"prefix":null,"suffix":null},"name":"comonad","version":"5.0.8"}, {"externalId":{"forge":"hackage","pieces":[hspec-core, 2.7.5],"prefix":null,"suffix":null},"name":"hspec-core","version":"2.7.5"}, {"externalId":{"forge":"hackage","pieces":[regex-base, 0.94.0.0],"prefix":null,"suffix":null},"name":"regex-base","version":"0.94.0.0"}, {"externalId":{"forge":"hackage","pieces":[cookie, 0.4.5],"prefix":null,"suffix":null},"name":"cookie","version":"0.4.5"}, {"externalId":{"forge":"hackage","pieces":[aeson-compat, 0.3.9],"prefix":null,"suffix":null},"name":"aeson-compat","version":"0.3.9"}, {"externalId":{"forge":"hackage","pieces":[Glob, 0.10.1],"prefix":null,"suffix":null},"name":"Glob","version":"0.10.1"}, {"externalId":{"forge":"hackage","pieces":[x509-validation, 1.6.11],"prefix":null,"suffix":null},"name":"x509-validation","version":"1.6.11"}, {"externalId":{"forge":"hackage","pieces":[parallel, 3.2.2.0],"prefix":null,"suffix":null},"name":"parallel","version":"3.2.2.0"}, {"externalId":{"forge":"hackage","pieces":[colour, 2.3.5],"prefix":null,"suffix":null},"name":"colour","version":"2.3.5"}, {"externalId":{"forge":"hackage","pieces":[zip, 1.5.0],"prefix":null,"suffix":null},"name":"zip","version":"1.5.0"}, {"externalId":{"forge":"hackage","pieces":[basement, 0.0.11],"prefix":null,"suffix":null},"name":"basement","version":"0.0.11"}, {"externalId":{"forge":"hackage","pieces":[cabal-doctest, 1.0.8],"prefix":null,"suffix":null},"name":"cabal-doctest","version":"1.0.8"}, {"externalId":{"forge":"hackage","pieces":[unbounded-delays, 0.1.1.1],"prefix":null,"suffix":null},"name":"unbounded-delays","version":"0.1.1.1"}, {"externalId":{"forge":"hackage","pieces":[byteable, 0.1.1],"prefix":null,"suffix":null},"name":"byteable","version":"0.1.1"}, {"externalId":{"forge":"hackage","pieces":[x509, 1.7.5],"prefix":null,"suffix":null},"name":"x509","version":"1.7.5"}, {"externalId":{"forge":"hackage","pieces":[asn1-encoding, 0.9.6],"prefix":null,"suffix":null},"name":"asn1-encoding","version":"0.9.6"}, {"externalId":{"forge":"hackage","pieces":[reflection, 2.1.6],"prefix":null,"suffix":null},"name":"reflection","version":"2.1.6"}, {"externalId":{"forge":"hackage","pieces":[refact, 0.3.0.2],"prefix":null,"suffix":null},"name":"refact","version":"0.3.0.2"}, {"externalId":{"forge":"hackage","pieces":[attoparsec-iso8601, 1.0.2.0],"prefix":null,"suffix":null},"name":"attoparsec-iso8601","version":"1.0.2.0"}, {"externalId":{"forge":"hackage","pieces":[mono-traversable, 1.0.15.1],"prefix":null,"suffix":null},"name":"mono-traversable","version":"1.0.15.1"}, {"externalId":{"forge":"hackage","pieces":[data-default-instances-old-locale, 0.0.1],"prefix":null,"suffix":null},"name":"data-default-instances-old-locale","version":"0.0.1"}, {"externalId":{"forge":"hackage","pieces":[streaming-commons, 0.2.2.1],"prefix":null,"suffix":null},"name":"streaming-commons","version":"0.2.2.1"}, {"externalId":{"forge":"hackage","pieces":[base-compat, 0.11.2],"prefix":null,"suffix":null},"name":"base-compat","version":"0.11.2"}, {"externalId":{"forge":"hackage","pieces":[semigroups, 0.19.1],"prefix":null,"suffix":null},"name":"semigroups","version":"0.19.1"}, {"externalId":{"forge":"hackage","pieces":[data-default-instances-containers, 0.0.1],"prefix":null,"suffix":null},"name":"data-default-instances-containers","version":"0.0.1"}, {"externalId":{"forge":"hackage","pieces":[distributive, 0.6.2.1],"prefix":null,"suffix":null},"name":"distributive","version":"0.6.2.1"}, {"externalId":{"forge":"hackage","pieces":[time-locale-compat, 0.1.1.5],"prefix":null,"suffix":null},"name":"time-locale-compat","version":"0.1.1.5"}, {"externalId":{"forge":"hackage","pieces":[invariant, 0.5.4],"prefix":null,"suffix":null},"name":"invariant","version":"0.5.4"}, {"externalId":{"forge":"hackage","pieces":[haskell-src, 1.0.3.1],"prefix":null,"suffix":null},"name":"haskell-src","version":"1.0.3.1"}, {"externalId":{"forge":"hackage","pieces":[foldl, 1.4.10],"prefix":null,"suffix":null},"name":"foldl","version":"1.4.10"}, {"externalId":{"forge":"hackage","pieces":[quickcheck-instances, 0.3.25.2],"prefix":null,"suffix":null},"name":"quickcheck-instances","version":"0.3.25.2"}, {"externalId":{"forge":"hackage","pieces":[insert-ordered-containers, 0.2.3.1],"prefix":null,"suffix":null},"name":"insert-ordered-containers","version":"0.2.3.1"}, {"externalId":{"forge":"hackage","pieces":[HUnit, 1.6.1.0],"prefix":null,"suffix":null},"name":"HUnit","version":"1.6.1.0"}, {"externalId":{"forge":"hackage","pieces":[adjunctions, 4.4],"prefix":null,"suffix":null},"name":"adjunctions","version":"4.4"}, {"externalId":{"forge":"hackage","pieces":[neat-interpolation, 0.5.1.2],"prefix":null,"suffix":null},"name":"neat-interpolation","version":"0.5.1.2"}, {"externalId":{"forge":"hackage","pieces":[indexed-traversable, 0.1.1],"prefix":null,"suffix":null},"name":"indexed-traversable","version":"0.1.1"}, {"externalId":{"forge":"hackage","pieces":[tf-random, 0.5],"prefix":null,"suffix":null},"name":"tf-random","version":"0.5"}, {"externalId":{"forge":"hackage","pieces":[swagger2, 2.6],"prefix":null,"suffix":null},"name":"swagger2","version":"2.6"}, {"externalId":{"forge":"hackage","pieces":[vector-algorithms, 0.8.0.4],"prefix":null,"suffix":null},"name":"vector-algorithms","version":"0.8.0.4"}, {"externalId":{"forge":"hackage","pieces":[hpp, 0.6.3],"prefix":null,"suffix":null},"name":"hpp","version":"0.6.3"}, {"externalId":{"forge":"hackage","pieces":[optparse-generic, 1.3.1],"prefix":null,"suffix":null},"name":"optparse-generic","version":"1.3.1"}, {"externalId":{"forge":"hackage","pieces":[asn1-parse, 0.9.5],"prefix":null,"suffix":null},"name":"asn1-parse","version":"0.9.5"}, {"externalId":{"forge":"hackage","pieces":[assoc, 1.0.2],"prefix":null,"suffix":null},"name":"assoc","version":"1.0.2"}, {"externalId":{"forge":"hackage","pieces":[prettyprinter-ansi-terminal, 1.1.2],"prefix":null,"suffix":null},"name":"prettyprinter-ansi-terminal","version":"1.1.2"}, {"externalId":{"forge":"hackage","pieces":[system-fileio, 0.3.16.4],"prefix":null,"suffix":null},"name":"system-fileio","version":"0.3.16.4"}, {"externalId":{"forge":"hackage","pieces":[heaps, 0.3.6.1],"prefix":null,"suffix":null},"name":"heaps","version":"0.3.6.1"}, {"externalId":{"forge":"hackage","pieces":[network-bsd, 2.8.1.0],"prefix":null,"suffix":null},"name":"network-bsd","version":"2.8.1.0"}, {"externalId":{"forge":"hackage","pieces":[pem, 0.2.4],"prefix":null,"suffix":null},"name":"pem","version":"0.2.4"}, {"externalId":{"forge":"hackage","pieces":[old-locale, 1.0.0.7],"prefix":null,"suffix":null},"name":"old-locale","version":"1.0.0.7"}, {"externalId":{"forge":"hackage","pieces":[quickcheck-io, 0.2.0],"prefix":null,"suffix":null},"name":"quickcheck-io","version":"0.2.0"}, {"externalId":{"forge":"hackage","pieces":[parsers, 0.12.10],"prefix":null,"suffix":null},"name":"parsers","version":"0.12.10"}, {"externalId":{"forge":"hackage","pieces":[managed, 1.0.8],"prefix":null,"suffix":null},"name":"managed","version":"1.0.8"}, {"externalId":{"forge":"hackage","pieces":[language-c, 0.8.3],"prefix":null,"suffix":null},"name":"language-c","version":"0.8.3"}, {"externalId":{"forge":"hackage","pieces":[data-diverse, 4.7.0.0],"prefix":null,"suffix":null},"name":"data-diverse","version":"4.7.0.0"}, {"externalId":{"forge":"hackage","pieces":[splitmix, 0.1.0.3],"prefix":null,"suffix":null},"name":"splitmix","version":"0.1.0.3"}, {"externalId":{"forge":"hackage","pieces":[setenv, 0.1.1.3],"prefix":null,"suffix":null},"name":"setenv","version":"0.1.1.3"}, {"externalId":{"forge":"hackage","pieces":[old-time, 1.1.0.3],"prefix":null,"suffix":null},"name":"old-time","version":"1.1.0.3"}, {"externalId":{"forge":"hackage","pieces":[range-set-list, 0.1.3.1],"prefix":null,"suffix":null},"name":"range-set-list","version":"0.1.3.1"}, {"externalId":{"forge":"hackage","pieces":[StateVar, 1.2.1],"prefix":null,"suffix":null},"name":"StateVar","version":"1.2.1"}, {"externalId":{"forge":"hackage","pieces":[optional-args, 1.0.2],"prefix":null,"suffix":null},"name":"optional-args","version":"1.0.2"}, {"externalId":{"forge":"hackage","pieces":[fuzzy, 0.1.0.0],"prefix":null,"suffix":null},"name":"fuzzy","version":"0.1.0.0"}, {"externalId":{"forge":"hackage","pieces":[type-equality, 1],"prefix":null,"suffix":null},"name":"type-equality","version":"1"}, {"externalId":{"forge":"hackage","pieces":[hostname, 1.0],"prefix":null,"suffix":null},"name":"hostname","version":"1.0"}, {"externalId":{"forge":"hackage","pieces":[polyparse, 1.13],"prefix":null,"suffix":null},"name":"polyparse","version":"1.13"}, {"externalId":{"forge":"hackage","pieces":[haddock-library, 1.9.0],"prefix":null,"suffix":null},"name":"haddock-library","version":"1.9.0"}, {"externalId":{"forge":"hackage","pieces":[optics-th, 0.3.0.2],"prefix":null,"suffix":null},"name":"optics-th","version":"0.3.0.2"}, {"externalId":{"forge":"hackage","pieces":[http-media, 0.8.0.0],"prefix":null,"suffix":null},"name":"http-media","version":"0.8.0.0"}, {"externalId":{"forge":"hackage","pieces":[js-dgtable, 0.5.2],"prefix":null,"suffix":null},"name":"js-dgtable","version":"0.5.2"}, {"externalId":{"forge":"hackage","pieces":[optics-core, 0.3.0.1],"prefix":null,"suffix":null},"name":"optics-core","version":"0.3.0.1"}, {"externalId":{"forge":"hackage","pieces":[generics-sop, 0.5.1.0],"prefix":null,"suffix":null},"name":"generics-sop","version":"0.5.1.0"}, {"externalId":{"forge":"hackage","pieces":[optics-extra, 0.3],"prefix":null,"suffix":null},"name":"optics-extra","version":"0.3"}, {"externalId":{"forge":"hackage","pieces":[void, 0.7.3],"prefix":null,"suffix":null},"name":"void","version":"0.7.3"}, {"externalId":{"forge":"hackage","pieces":[js-flot, 0.8.3],"prefix":null,"suffix":null},"name":"js-flot","version":"0.8.3"}, {"externalId":{"forge":"hackage","pieces":[js-jquery, 3.3.1],"prefix":null,"suffix":null},"name":"js-jquery","version":"3.3.1"}, {"externalId":{"forge":"hackage","pieces":[mwc-random, 0.14.0.0],"prefix":null,"suffix":null},"name":"mwc-random","version":"0.14.0.0"}, {"externalId":{"forge":"hackage","pieces":[charset, 0.3.7.1],"prefix":null,"suffix":null},"name":"charset","version":"0.3.7.1"}, {"externalId":{"forge":"hackage","pieces":[monoid-subclasses, 1.0.1],"prefix":null,"suffix":null},"name":"monoid-subclasses","version":"1.0.1"}, {"externalId":{"forge":"hackage","pieces":[Only, 0.1],"prefix":null,"suffix":null},"name":"Only","version":"0.1"}, {"externalId":{"forge":"hackage","pieces":[math-functions, 0.3.4.1],"prefix":null,"suffix":null},"name":"math-functions","version":"0.3.4.1"}, {"externalId":{"forge":"hackage","pieces":[sop-core, 0.5.0.1],"prefix":null,"suffix":null},"name":"sop-core","version":"0.5.0.1"}, {"externalId":{"forge":"hackage","pieces":[indexed-profunctors, 0.1],"prefix":null,"suffix":null},"name":"indexed-profunctors","version":"0.1"}, {"externalId":{"forge":"hackage","pieces":[primes, 0.2.1.0],"prefix":null,"suffix":null},"name":"primes","version":"0.2.1.0"}]```


and the references for each

```
2021-01-29T06:07:11.0743220Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "type": "LABEL_LIST",
2021-01-29T06:07:11.0749381Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "stringListValue": ["@stackage//:conduit-1.3.4/ChangeLog.md", "@stackage//:conduit-1.3.4/LICENSE", "@stackage//:conduit-1.3.4/README.md", "@stackage//:conduit-1.3.4/Setup.lhs", "@stackage//:conduit-1.3.4/benchmarks/optimize-201408.hs", "@stackage//:conduit-1.3.4/benchmarks/unfused.hs", "@stackage//:conduit-1.3.4/conduit.cabal", "@stackage//:conduit-1.3.4/fusion-macros.h", "@stackage//:conduit-1.3.4/src/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators/Stream.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators/Unqualified.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Fusion.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/List/Stream.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Pipe.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Lift.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/List.hs", "@stackage//:conduit-1.3.4/src/Data/Streaming/FileRead.hs", "@stackage//:conduit-1.3.4/src/Data/Streaming/Filesystem.hs", "@stackage//:conduit-1.3.4/src/System/Win32File.hsc", "@stackage//:conduit-1.3.4/test/Data/Conduit/Extra/ZipConduitSpec.hs", "@stackage//:conduit-1.3.4/test/Data/Conduit/StreamSpec.hs", "@stackage//:conduit-1.3.4/test/Spec.hs", "@stackage//:conduit-1.3.4/test/StreamSpec.hs", "@stackage//:conduit-1.3.4/test/doctests.hs", "@stackage//:conduit-1.3.4/test/main.hs", "@stackage//:conduit-1.3.4/test/subdir/dummyfile.txt"],
```

```
"stringListValue": ["@stackage//:conduit-extra-1.3.5/ChangeLog.md", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Attoparsec.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Binary.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/ByteString/Builder.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Filesystem.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Foldl.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Lazy.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Network.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Network/UDP.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Network/Unix.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Process.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Process/Typed.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Text.hs", "@stackage//:conduit-extra-1.3.5/Data/Conduit/Zlib.hs", "@stackage//:conduit-extra-1.3.5/LICENSE", "@stackage//:conduit-extra-1.3.5/README.md", "@stackage//:conduit-extra-1.3.5/Setup.lhs", "@stackage//:conduit-extra-1.3.5/bench/blaze.hs", "@stackage//:conduit-extra-1.3.5/conduit-extra.cabal", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/AttoparsecSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/BinarySpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/ByteString/BuilderSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/ExtraSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/FilesystemSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/LazySpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/NetworkSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/Process/TypedSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/ProcessSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/TextSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Data/Conduit/ZlibSpec.hs", "@stackage//:conduit-extra-1.3.5/test/Spec.hs", "@stackage//:conduit-extra-1.3.5/test/filesystem/bar.txt", "@stackage//:conduit-extra-1.3.5/test/filesystem/baz.txt", "@stackage//:conduit-extra-1.3.5/test/filesystem/bin/bin.txt", "@stackage//:conduit-extra-1.3.5/test/filesystem/foo.txt", "@stackage//:conduit-extra-1.3.5/test/random"],```

```
2021-01-29T06:07:11.0805764Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---         "ruleInput": ["@stackage//:base", "@stackage//:bytestring", "@stackage//:conduit-1.3.4/ChangeLog.md", "@stackage//:conduit-1.3.4/LICENSE", "@stackage//:conduit-1.3.4/README.md", "@stackage//:conduit-1.3.4/Setup.lhs", "@stackage//:conduit-1.3.4/benchmarks/optimize-201408.hs", "@stackage//:conduit-1.3.4/benchmarks/unfused.hs", "@stackage//:conduit-1.3.4/conduit.cabal", "@stackage//:conduit-1.3.4/fusion-macros.h", "@stackage//:conduit-1.3.4/src/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators/Stream.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Combinators/Unqualified.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Conduit.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Fusion.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/List/Stream.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Internal/Pipe.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/Lift.hs", "@stackage//:conduit-1.3.4/src/Data/Conduit/List.hs", "@stackage//:conduit-1.3.4/src/Data/Streaming/FileRead.hs", "@stackage//:conduit-1.3.4/src/Data/Streaming/Filesystem.hs", "@stackage//:conduit-1.3.4/src/System/Win32File.hsc", "@stackage//:conduit-1.3.4/test/Data/Conduit/Extra/ZipConduitSpec.hs", "@stackage//:conduit-1.3.4/test/Data/Conduit/StreamSpec.hs", "@stackage//:conduit-1.3.4/test/Spec.hs", "@stackage//:conduit-1.3.4/test/StreamSpec.hs", "@stackage//:conduit-1.3.4/test/doctests.hs", "@stackage//:conduit-1.3.4/test/main.hs", "@stackage//:conduit-1.3.4/test/subdir/dummyfile.txt", "@stackage//:directory", "@stackage//:exceptions", "@stackage//:filepath", "@stackage//:mono-traversable", "@stackage//:mtl", "@stackage//:primitive", "@stackage//:resourcet", "@stackage//:text", "@stackage//:transformers", "@stackage//:unix", "@stackage//:unliftio-core", ```

```2021-01-29T06:07:11.3983217Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "stringListValue": ["@stackage//:tar-conduit-0.3.2/ChangeLog.md", "@stackage//:tar-conduit-0.3.2/LICENSE", "@stackage//:tar-conduit-0.3.2/README.md", "@stackage//:tar-conduit-0.3.2/Setup.hs", "@stackage//:tar-conduit-0.3.2/src/Data/Conduit/Tar.hs", "@stackage//:tar-conduit-0.3.2/src/Data/Conduit/Tar/Types.hs", "@stackage//:tar-conduit-0.3.2/src/Data/Conduit/Tar/Unix.hs", "@stackage//:tar-conduit-0.3.2/src/Data/Conduit/Tar/Windows.hs", "@stackage//:tar-conduit-0.3.2/tar-conduit.cabal", "@stackage//:tar-conduit-0.3.2/tests/Space.hs", "@stackage//:tar-conduit-0.3.2/tests/Spec.hs", "@stackage//:tar-conduit-0.3.2/tests/Time.hs", "@stackage//:tar-conduit-0.3.2/tests/files/libpq-0.3.tar.gz", "@stackage//:tar-conduit-0.3.2/tests/files/subdir.tar"],```

```2021-01-29T06:07:11.8540391Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "stringListValue": ["@stackage//:xml-conduit-1.9.0.0/ChangeLog.md", "@stackage//:xml-conduit-1.9.0.0/LICENSE", "@stackage//:xml-conduit-1.9.0.0/README.md", "@stackage//:xml-conduit-1.9.0.0/Setup.lhs", "@stackage//:xml-conduit-1.9.0.0/Text/XML.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Cursor.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Cursor/Generic.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Stream/Parse.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Stream/Render.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Stream/Token.hs", "@stackage//:xml-conduit-1.9.0.0/Text/XML/Unresolved.hs", "@stackage//:xml-conduit-1.9.0.0/test/doctest.hs", "@stackage//:xml-conduit-1.9.0.0/test/unit.hs", "@stackage//:xml-conduit-1.9.0.0/xml-conduit.cabal"],
```
```2021-01-29T06:07:12.8735981Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "stringListValue": ["@stackage//:http-conduit-2.3.7.4/ChangeLog.md", "@stackage//:http-conduit-2.3.7.4/LICENSE", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Client/Conduit.hs", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Conduit.hs", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Simple.hs", "@stackage//:http-conduit-2.3.7.4/README.md", "@stackage//:http-conduit-2.3.7.4/Setup.lhs", "@stackage//:http-conduit-2.3.7.4/certificate.pem", "@stackage//:http-conduit-2.3.7.4/http-conduit.cabal", "@stackage//:http-conduit-2.3.7.4/key.pem", "@stackage//:http-conduit-2.3.7.4/multipart-example.bin", "@stackage//:http-conduit-2.3.7.4/nyan.gif", "@stackage//:http-conduit-2.3.7.4/test/CookieTest.hs", "@stackage//:http-conduit-2.3.7.4/test/main.hs"],```

```2021-01-29T06:07:12.8766299Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---         "ruleInput": ["@stackage//:aeson", "@stackage//:attoparsec", "@stackage//:base", "@stackage//:bytestring", "@stackage//:conduit", "@stackage//:conduit-extra", "@stackage//:http-client", "@stackage//:http-client-tls", "@stackage//:http-conduit-2.3.7.4/ChangeLog.md", "@stackage//:http-conduit-2.3.7.4/LICENSE", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Client/Conduit.hs", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Conduit.hs", "@stackage//:http-conduit-2.3.7.4/Network/HTTP/Simple.hs", "@stackage//:http-conduit-2.3.7.4/README.md", "@stackage//:http-conduit-2.3.7.4/Setup.lhs", "@stackage//:http-conduit-2.3.7.4/certificate.pem", "@stackage//:http-conduit-2.3.7.4/http-conduit.cabal", "@stackage//:http-conduit-2.3.7.4/key.pem", "@stackage//:http-conduit-2.3.7.4/multipart-example.bin", "@stackage//:http-conduit-2.3.7.4/nyan.gif", "@stackage//:http-conduit-2.3.7.4/test/CookieTest.hs", "@stackage//:http-conduit-2.3.7.4/test/main.hs", "@stackage//:http-types", "@stackage//:mtl", "@stackage//:resourcet", "@stackage//:transformers", "@stackage//:unliftio-core"]```

```
2021-01-29T06:07:13.4374064Z 2021-01-29 06:07:10 DEBUG [main-Executable_Stream_Thread] ---           "stringListValue": ["@stackage//:cereal-conduit-0.8.0/ChangeLog.md", "@stackage//:cereal-conduit-0.8.0/Data/Conduit/Cereal.hs", "@stackage//:cereal-conduit-0.8.0/Data/Conduit/Cereal/Internal.hs", "@stackage//:cereal-conduit-0.8.0/LICENSE", "@stackage//:cereal-conduit-0.8.0/README.md", "@stackage//:cereal-conduit-0.8.0/Setup.lhs", "@stackage//:cereal-conduit-0.8.0/Test/Main.hs", "@stackage//:cereal-conduit-0.8.0/cereal-conduit.cabal"],```